target/linux/airoha/dts/en7581.dtsi

```diff
@@ -146,12 +146,58 @@
 		};
 
 		uart1: serial@1fbf0000 {
-			compatible = "ns16550";
+			compatible = "airoha,en7523-uart";
 			reg = <0x0 0x1fbf0000 0x0 0x30>;
 			reg-io-width = <4>;
 			reg-shift = <2>;
 			interrupts = <GIC_SPI 18 IRQ_TYPE_LEVEL_HIGH>;
 			clock-frequency = <1843200>;
+
+			status = "okay";
```

Comment:
@merbanan I guess "okay" it is the default one

Reply:
Adressed.